### PR TITLE
fix(hogql): resize query + tooltip bounds

### PR DESCRIPTION
**Pull request by mariusandra**
## Problem

This was getting frustrating

## How did you test this code?

👀

### CODE REVIEW

File: frontend/src/queries/nodes/HogQLQuery/HogQLQueryEditor.tsx [hunk 1]
```diff
@@ -44,7 +44,8 @@ export function HogQLQueryEditor(props: HogQLQueryEditorProps): JSX.Element {
         <div className="space-y-2">
             <div
                 data-attr="hogql-query-editor"
-                className={'flex flex-col p-2 border rounded bg-bg-light space-y-2 resize-y w-full'}
+                className={'flex flex-col p-2 border rounded bg-bg-light space-y-2 resize-y w-full overflow-hidden'}
+                style={{ height: 318 }}
```
Comment (Twixes) [inline frontend/src/queries/nodes/HogQLQuery/HogQLQueryEditor.tsx:48]: Why this height in particular?

**Review by Twixes — COMMENTED** [review]
Big +1

File: frontend/src/queries/nodes/HogQLQuery/HogQLQueryEditor.tsx (hunk 1 above)
Comment (mariusandra) [reply frontend/src/queries/nodes/HogQLQuery/HogQLQueryEditor.tsx:48]: Oh... I didn't have the AI bar enabled. It does fit the default query if that's not there

Comment (Twixes) [reply frontend/src/queries/nodes/HogQLQuery/HogQLQueryEditor.tsx:48]: Makes sense, since it's on just for us now

**Pull request merged**